de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ASwtBotTestCase.java

```diff
@@ -161,6 +168,77 @@ protected SWTBotTreeItem openEditor(SWTBotTreeItem item) {
 		return newItem;
 	}
 	
+	/**
+	 * @param item Tree item for which a diagram editor gets opened
+	 * @return Graphiti GEF Editor handle that SWTBot can work on
+	 */
+	protected SWTBotGefEditor openDiagramEditorForTreeItem(SWTBotTreeItem item) {
+		SWTGefBot gefBot = new SWTGefBot();
+		item.contextMenu().menu("Open Diagram Editor").click();

```

Comment:
I am not a 100% sure if we can generalize this so easily.
The "Open Diagram Editor" handler in FDIR is a custom handler.

Reply:
I'll remove it for now until we come up with a better more generic way